src/servers/src/postgres/handler.rs

```diff
@@ -336,9 +352,23 @@ impl ExtendedQueryHandler for PostgresServerHandler {
     where
         C: ClientInfo + Unpin + Send + Sync,
     {
+        let _timer = timer!(
+            crate::metrics::METRIC_POSTGRES_QUERY_TIMER,
+            &[
+                (
+                    crate::metrics::METRIC_POSTGRES_SUBPROTOCOL_LABEL,
+                    crate::metrics::METRIC_POSTGRES_EXTENDED_QUERY
+                ),
+                (
+                    crate::metrics::METRIC_DB_LABEL,
+                    &self.query_ctx.get_db_string()
+                )
+            ]
+        );
```

Comment:
These metrics are great, but my concern is they might be hard to maintain (e.g., forget to some fields when adding new metrics or new interfaces). Unfortunately, I don't have any good solution 🥹

Reply:
@waynexia Yep, it always happens again and again. I have no good solutions too. I think the code review may help.